src/inmanta/ast/__init__.py

```diff
@@ -186,7 +186,7 @@ def resolve(self) -> Location:
 
 
 class TypeReferenceAnchor(Anchor):
-    def __init__(self, range: Range, namespace: "Namespace", type: str) -> None:
+    def __init__(self, range: Range, namespace: "Namespace", type: LocatableString) -> None:
         Anchor.__init__(self, range=range)
```

Comment:
do we still need the range if the type is locatable?

Reply:
Valid point, I'll look into it